ansible/lifecycle_hook.yml

```diff
@@ -0,0 +1,9 @@
+- name: Lifecycle Hook (fallback)
+  hosts: localhost
+  run_once: true
+  become: false
+  gather_facts: false
+  tasks:
+  - name: Placeholder
+    debug:
+      msg: "No custom lifecycle hook provided. This is the lifecycle hook placeholder"
```

Comment:
instead of creating an empty hook, can't we just use `/dev/null` ?  (i didn't try)

Reply:
I like having the output there. Makes it obvious in the log that no hook was called.